### PR TITLE
Add canvas labels to thumbnail navigation

### LINKED ISSUE #1932: ThumbnailNavigation styling, labels, and visual design
Our current implementation of ThumbnailNavigation is missing the styles in the mocks.

Needed things:
 - [ ] ~~Effective padding / margins for both single and book view~~
 - [x] Labels added
 - [x] Container areas should work w/ theming

Thumbnails navigation
- [x] use `<nav />` element, add `aria-label="Thumbnail carousel"` (#2101)

### PR DESCRIPTION
**Pull request by jkeck**
Part of #1932

thumbnail-buttons

### CODE REVIEW

File: src/containers/ThumbnailNavigation.js [hunk 1]
```diff
@@ -24,8 +25,22 @@ const mapDispatchToProps = {
   setCanvas: actions.setCanvas,
 };
 
+const styles = {
+  titleBar: {
+    background: 'linear-gradient(to top, rgba(0,0,0,0.7) 0%, rgba(0,0,0,0.3) 70%, rgba(0,0,0,0) 100%)',
+    textAlign: 'left',
+  },
+  title: {
+    color: 'white',
```
Comment (jkeck) [inline src/containers/ThumbnailNavigation.js]: Is there somewhere I should be getting this color from instead?

**Comment by jvine** [comment]
So on further exploration... it looks like this is rendering invalid HTML. A `<ul>` element can only have `<li>` elements as direct children, and this has a bunch of `<div>`s in between.

**Comment by jkeck** [comment]
I was curious why I wasn't getting any errors from React (which has thrown errors about similar HTML structural issues).

I'm not sure what we can do if we want to use the library (`react-virtualized`) that we're using to handle lazily loading the canvas thumbs.

Would the addition of the button inside the div to give a clickable element be an acceptable improvement?  Otherwise, I can kill the list/button changes all-together and simply add the canvas label as a caption in the `GridListTileBar`.

**Comment by jkeck** [comment]
I've popped on a commit to remove the list but keep the button (now that we're wrapped in a `<nav />` as of #2070 being 🚢ed).  We can squash that down to one commit if it's 👍

File: src/containers/ThumbnailNavigation.js (hunk 1 above)
Comment (cbeer) [reply src/containers/ThumbnailNavigation.js]: `theme.palette.common.white`? there's also `theme.palette.getContrastText()`, which I don't understand but sounds promising.

**Comment by mejackreed** [comment]
This looks really great! I did notice one thing where in BookView the second thumb seems to be missing:

screen shot 2019-03-07 at 7 25 12 am

File: src/containers/ThumbnailNavigation.js (hunk 1 above)
Comment (jkeck) [reply src/containers/ThumbnailNavigation.js]: I tried this out, but whenever I use `theme` into my styles, everything goes haywire when I switch themes.  Maybe somebody can help me look at this today in the office (or if anybody know of some addl. step I need to do).

Comment (cbeer) [reply src/containers/ThumbnailNavigation.js]: ✋; sounds good

**Comment by jkeck** [comment]
Closed by #2088

**Pull request closed without merging**